release-schema.json

```diff
@@ -50,10 +50,38 @@
           "maximum": 1,
           "minimum": 0
         },
+        "competitiveMaximumPercentage": {
+          "title": "Maximum percentage of total value for competitive subcontracting",
+          "description": "Maximum percentage of the contract value the procuring entity asks the contractor to subcontract to third parties through a competitive procedure.",
+          "type": [
+            "number",
+            "null"
+          ],
+          "maximum": 1,
+          "minimum": 0.0001
```

Comment:
Instead of 0.0001, we can use `"exclusiveMinimum": true`. Same above.

Reply:
I get an error during validation when I add `exclusiveMinimum`:

> Additional properties are not allowed ('exclusiveMinimum' was unexpected) (properties/definitions/additionalProperties/properties/properties/additionalProperties/additionalProperties)

It also results in an error if add `exclusiveMinimum` without `minimum`.

Reply:
https://github.com/open-contracting-extensions/ocds_subcontracting_extension/pull/3/checks?check_run_id=554900208

Reply:
That's the last thing to fix in this PR.

Reply:
Thanks - it's because this was the first time we used that property, so I had to relax our checks.